modules/gitops-installing-argocd-cli-on-linux-using-rpm.adoc

```diff
@@ -0,0 +1,137 @@
+// Module is included in the following assemblies:
+//
+// * installing_gitops/installing-argocd-gitops-cli.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-installing-argocd-cli-on-linux-using-rpm"]
+= Installing the {gitops-title} CLI on Linux using an RPM
+
+For {op-system-base-full} version 8 or later, you can install the {gitops-shortname} `argocd` CLI as an RPM by using a package manager, such as `yum` or `dnf`. This allows the {gitops-shortname} `argocd` CLI version to be automatically managed by the system. For example, using a command like `dnf upgrade` upgrades all packages, including `argocd`, if a new version is available.
+
+.Prerequisites
+
+* You have an active {OCP} subscription on your Red Hat account.
+* You have root or `sudo` privileges on your local system.
+
+.Procedure
+
+. Register with Red Hat Subscription Manager:
```

Comment:
It might be worth noting that this should be run as root. The command prompt indicates these privileges, but it's generally worth explicitly stating, too.